### PR TITLE
Update readme install section for Rails 6 with Webpacker

### PR DESCRIPTION
**Pull request by quatauta**
Documentation update in README.md to add instructions for using `local_time` with Ruby on Rails 6 with Webpacker.

I struggled to find instructions on how to use `local_time` with Ruby on Rails 6 and Webpacker. Based on a [comment on Stack Overflow](https://stackoverflow.com/a/58404848), I figured how to do it and updated README.md.

I am happy to edit this PR to your requirements, and please feel free to update/delete as you like :-)

### CODE REVIEW

**Comment by quatauta** [comment]
May be duplicate of #99

File: README.md [hunk 1]
```diff
@@ -7,15 +7,36 @@ Local Time makes it easy to display times and dates to users in their local time
...
+           "dependencies": {
+             "local-time": "^2.1.0"
+           }
+         }
+         ```
+      1. Run `yarn install` to install npm packages
+         ```sh
+         $ yarn install
+         [...] Resolving packages ...
+         ```
```
Comment (adammiribyan) [inline README.md:35]: The two above can be combined into

```
yarn add local-time
```

Comment (quatauta) [reply README.md:27]: Thank you @adammiribyan ! I am new to yarn and did not yet know about yarn install :-) I updated the PR.

**Comment by josefarias** [comment]
https://github.com/basecamp/local_time/pull/139

**Pull request closed without merging**